README.md

```diff
@@ -1,64 +1,99 @@
 # Sample Project
 
-This is as an example on how teams can structure their project repositories. Thanks to Lindsey Heagey and Joachim Meyer for the template!
+This is an example of how teams can structure their project repositories and format their project README.md file.
 
-## Files
+When creating a project repository from this template, we recommend choosing "Public" (so other participants can follow progress). You can also select Private initially and make Public before final presentation. We also recommend adding a "topic" to your repository details (click on the gear icon next to the "About" section on the repository page) to help others find your work (e.g. `uw-hackweek-snowex2024`).
 
+
+## Files and folders in your project repository
+
+This template provides the following suggested organizaiton structure for the project repository, but each project team is free to organize their repository as they see fit.
+
+* **`contributors/`**
+<br> Each team member can create their own folder under contributors, within which they can work on their own scripts, notebooks, and other files. Having a dedicated folder for each person helps to prevent conflicts when merging with the main branch. This is a good place for team members to start off exploring data and methods for the project.
+* **`notebooks/`**
+<br> Notebooks that are considered delivered results for the project should go in here.
+* **`scripts/`**
+<br> Helper utilities that are shared with the team should go in here.
```

Comment:
I wonder if people might think "helper" refers to our hackweek role as helper instead of a script?

It's difficult to say this succinctly but I'm thinking something like: "This is a place to store functions or subroutines that get called by multiple people. These will be files other than Jupyter Notebooks such as Python scripts (.py)."

Reply:
Thanks! I think I have seen it used when project leaders bring in their own scripts (e.g. for data access or pre-processing steps) to share with the team. I've updated the text to try and clarify.